sql/core/src/main/java/org/apache/spark/sql/sources/v2/MicroBatchReadSupportProvider.java

```diff
@@ -45,8 +51,8 @@
    * @param options the options for the returned data source reader, which is an immutable
    *                case-insensitive string-to-string map.
    */
-  MicroBatchReader createMicroBatchReader(
-      Optional<StructType> schema,
-      String checkpointLocation,
-      DataSourceOptions options);
+  MicroBatchReadSupport createMicroBatchReadSupport(
+    Optional<StructType> schema,
```

Comment:
Nit: whitespace change that doesn't fit with typical Java style of 2 indent levels for method params.